app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -628,14 +628,26 @@
 
                 </LinearLayout>
 
+                <Button
+                    android:id="@+id/buttonNewFrontImage"
+                    style="@style/ButtonBorder"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_horizontal"
+                    android:layout_marginStart="@dimen/spacing_normal"
+                    android:layout_marginEnd="@dimen/spacing_normal"
+                    android:text="@string/update_image"
```

Comment:
send_updated_image ?

Reply:
<string name="send_updated_image">Send an Updated Image</string>
    <string name="update_image">Update image</string>
<string name="take_extra_packaging_pictures">Take extra photos of packaging</string>

Reply:
it's the same meaning, so no big deal

Reply:
Yeah, was confused a bit if would be ok. No problem, I will revert adding of this new resource.